byterun/caml/sys.h

```diff
@@ -18,6 +18,10 @@
 
 #include "misc.h"
 
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Consistently rejected ? Have a look at `caml/alloc.h` for example and you will see that all public headers actually have it. So, your question is probably: why didn't this PR add it to all the other headers ?